packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -375,6 +375,20 @@ export class DeltaManager
         return this.clientSequenceNumberObserved < (this.clientSequenceNumber - this.trailingNoopCount);
     }
 
+    public connectionProps(): ITelemetryProperties {
+        if (this.connection !== undefined) {
+            return {
+                sequenceNumber: this.lastSequenceNumber,
+                connectionMode: this.connectionMode,
+            };
+        } else {
+            return {
+                // Report how many ops this client sent in just disconnected session
```

Comment:
I think you want to say "last disconnected session" instead of "just disconnected session"?

Reply:
or maybe previous?

Reply:
changed